newtab.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>banksy</title>
+  <script src="app.js"></script>
+</head>
```

Comment:
<script defer src="js/main.js"></script>
<link rel="stylesheet" href="css/main.css"></link>
something like that.